### PR TITLE
pattern/cultural-change-through-hiring

### PR DESCRIPTION
**Pull request by nyeates**
Innersource projects have an issue getting enough employees who have natively inner source / open source abilities. Company has trouble attracting developers who are interested in open source and inner source.

### CODE REVIEW

**Comment by spier** [comment]
Hi @nyeates. Sorry for bugging you.

Do you have time and want to push this PR forward?

If yes, then I would suggest to do one pass to adapt it to the [Pattern Template](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/meta/pattern-template.md).

If no, then I would suggest to close this PR.

Thanks for your help.

**Comment by lenucksi** [comment]
Maybe this could be of interest to @BrittanyIstenes too?

**Comment by spier** [comment]
This PR seems to be linked to issue #37 .

File: patterns/1-initial/cultural-change-through-hiring.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+> What makes the problem difficult? What are the trade-offs? These are constraints that **can be changed** at a cost. The solution might change one or more of these forces in order to solve the problem, while also in-turn changing the context.
+
+* InnerSource projects have a hard time finding employees who already know InnerSource or Open Source methodologies.
+* InnerSource as a concept is not well known
+
+## Solutions
+
+* Engineering and HR make a concerted effort to identify the skills that they want to recruit for in new employees and train the existing staff in
+* A handful of key-hires need to be identified, to act as evangelists for the InnerSource way of working
+* ...
```
Comment (spier) [inline patterns/1-initial/cultural-change-through-hiring.md]:
```suggestion
```

**Review by spier — APPROVED** [review]
Approving this PR on the premise that:
a) the PR is so old that it is unlikely that the original authors will still work on it and
b) it is better to have a half-finished pattern draft in the main branch, than in a PR, as readers are more likely to find it in the main branch

**Pull request merged**